src/network/networkd-link.h

```diff
@@ -97,6 +99,8 @@ typedef struct Link {
         bool ipv4ll_address:1;
         bool ipv4ll_route:1;
 
+        bool neighbors_configured;
+
```

Comment:
why the empty line, it makes sense to keep this together with the other variables `static_routes_configured` and such

Reply:
Since configuring neighbors is its own stage it looks like it should be separate. The static_routes_configured is separate from dhcp{4,6}_configured and ndisc_configured is in its own section.